### PR TITLE
adjust style of markdown editor

### LINKED ISSUE #132: Markdown editor style
The markdown editor produces a few weird artifacts:
bildschirmfoto 2015-05-05 um 16 22 31
The label "Text" has a red font and on the top and bottom of the textarea there's a ``border: 1px dashed red``.

### PR DESCRIPTION
**Pull request by Nef10**
fix #132 
@invliD please have a look

### CODE REVIEW

File: _1327/static/less/1327.less [hunk 1]
```diff
@@ -226,3 +226,17 @@ footer a {
...
+	box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+}
+
+.md-editor {
+	border-radius: 4px;
+}
+
+.md-editor textarea {
+	padding: 6px 12px
+}
```
Comment (invliD) [inline _1327/static/less/1327.less]: Please add a newline.

File: _1327/static/less/1327.less [hunk 2]
```diff
@@ -226,3 +226,17 @@ footer a {
...
 }
+
+.has-error .md-editor {
+	border-color: #b1063a;
+	-webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+	box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+}
+
+.md-editor {
+	border-radius: 4px;
```
Comment (invliD) [inline _1327/static/less/1327.less]: Use bootstrap variables for all values.

**Comment by invliD** [comment]
The entire editor still has a red border and "Text" is also still red. Why does the form control have the bootstrap class ``has-error`` when creating a new document?
The textarea also still has a dashed border on top and on the bottom, it's just gray now.

**Comment by Nef10** [comment]
Because it is a mandatory field, I did not changed this.
The dashed border is the style of the editor (see http://www.codingdrama.com/bootstrap-markdown/)

**Comment by invliD** [comment]
But it's already red when you open the page, not just after submitting without text.
I think you should remove the dashed border. It looks kinda weird.

**Comment by Nef10** [comment]
The problem is caused by the form validation and not by my styles.

**Comment by Nef10** [comment]
But another problem: we want to style the div with the editor and not textarea itself on focus. Unfortunately this requires javascript because the css selector is not supported yet: http://css4-selectors.com/selector/css4/subject-of-selector-with-child-combinator/

File: _1327/static/less/1327.less [hunk 3]
```diff
@@ -226,3 +226,17 @@ footer a {
 table.table-narrow {
 	max-width: 400px;
 }
+
+.has-error .md-editor {
+  .form-control-validation(@state-danger-text; @state-danger-text; @state-danger-bg);
```
Comment (invliD) [inline _1327/static/less/1327.less]: What does this do? I can't tell the difference between using this and commenting it out.

**Comment by Nef10** [comment]
Regarding error when creating a new document see #148.
Finished now

**Comment by invliD** [comment]
Please rebase on top of master instead of merging master to your feature branch.

**Pull request merged**